src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Offender.kt

```diff
@@ -62,7 +62,9 @@ class OffenderBuilder(
   private val repository: OffenderBuilderRepository,
   private val bookingBuilderFactory: BookingBuilderFactory,
 ) : OffenderDsl {
-  private lateinit var offender: Offender
+  private lateinit var rootOffender: Offender
+  private lateinit var aliasOffender: Offender
```

Comment:
I was kind of expecting this to be a list of aliases rather than a single one given you can call the `alias` method multiple times.

Reply:
Maybe its poorly named, its intended as the 'current' offender, used by the booking function to hook up its offender parent. When Ive reorganised things to use an alias dsl this may go